cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/KafkaClusterState.java

```diff
@@ -52,7 +53,9 @@ public KafkaClusterState(Cluster kafkaCluster, KafkaCruiseControlConfig config)
 
   private String getJSONString(CruiseControlParameters parameters) {
     Gson gson = new Gson();
-    Map<String, Object> jsonStructure = getJsonStructure(((KafkaClusterStateParameters) parameters).isVerbose());
+    boolean isVerbose = ((KafkaClusterStateParameters) parameters).isVerbose();
```

Comment:
nit: in this method and in `getPlaintext` we do cast form `CruiseControlParameters` to `KafkaClusterStateParameters` couple of times, maybe we can save one redundant cast.